js/screens/EmergencyScreen.js

```diff
@@ -11,202 +12,152 @@ import PropTypes from 'prop-types';
     (state) => state.settings.hotlineNumber
   );
-  const savedAttorneyName = useSelector((state) => state.settings.attorneyName);
-  const savedAttorneyNumber = useSelector(
-    (state) => state.settings.attorneyNumber
-  );
   const phoneNum = {
     number: savedHotlineNumber,
     prompt: false,
   };
   return (
-    <View style={{ flex: 1 }}>
+    <View style={{ backgroundColor: 'white', flex: 1 }}>
       <SafeAreaView style={{ flex: 1 }}>
-        <ScrollView contentContainerStyle={{ flexGrow: 1 }} style={{ flex: 1 }}>
-          <TouchableOpacity
-            onPress={navigation.goBack}
-            style={{ alignSelf: 'flex-start', paddingLeft: 20, paddingTop: 12 }}
-          >
-            <FireIcon
-              color={colors.charcoalGrey}
-              name={ICON_NAMES.CLOSE}
-              size={32}
-            />
-          </TouchableOpacity>
-          <View style={{ flexGrow: 1 }} />
-          <View style={{ paddingHorizontal: 56 }}>
-            <View
-              style={{
-                alignItems: 'center',
-                flexDirection: 'row',
-                justifyContent: 'center',
-              }}
-            >
+        <ScrollView
+          contentContainerStyle={{
+            flexGrow: 1,
+            justifyContent: 'space-between',
+          }}
+          style={{ flex: 1 }}
+        >
+          <View>
+            <TouchableOpacity onPress={navigation.goBack} style={styles.close}>
               <FireIcon
-                color={colors.primary}
-                name={ICON_NAMES.ALERT}
+                color={colors.charcoalGrey}
+                name={ICON_NAMES.CLOSE}
                 size={32}
-                style={{ paddingRight: 6, paddingTop: 3 }}
               />
-              <Text style={textStyles.h1}>{t('emergency_toolkit')}</Text>
-            </View>
-            <View style={{ paddingTop: 12 }}>
+            </TouchableOpacity>
+            <TitleSection />
+            <InfoSection />
+            <View style={{ paddingHorizontal: 56, paddingTop: 12 }}>
               <PrimaryButton
                 darkMode
                 onPress={() => call(phoneNum)}
                 title={t('call_emergency_hotline')}
               />
-              <PrimaryButton
-                darkMode
-                onPress={() => setRightsCardVisible(!RightsCardVisible)}
-                title={t('rights_card')}
-              />
             </View>
           </View>
-          <View style={{ flexGrow: 1, maxHeight: 100 }} />
           <HelpButton
             centered
             onPress={() => setInfoModalVisible(!InfoModalVisible)}
-            title={t('rights_card_what_is_this')}
+            title={t('learn_more')}
           />
-          <View style={{ flexGrow: 1 }} />
         </ScrollView>
-
-        {/* ABSOLUTE MODALS */}
+        {/* MODAL */}
         <InfoModal
           isVisible={InfoModalVisible}
           setModalVisible={setInfoModalVisible}
```

Comment:
```suggestion
          onClose={() => setInfoModalVisible(false)}
```
(See above comment on the `InfoModal`)